--- tr_pipelinec.gen.c.BAK	2022-05-09 16:17:50.123074680 -0300
+++ tr_pipelinec.gen.c	2022-05-09 16:25:11.258879723 -0300
@@ -172,2 +172,3 @@
 typedef struct full_state_t { scene_t scene; color gold_color; color gold_reflect_color; color lava_color; coord_type plane_y; coord_type sphere_x; coord_type sphere_z; coord_type sphere_y; color_type heat; coord_type camera_y; coord_type camera_z; coord_type plane_x; coord_type sphere_xvel; coord_type sphere_yvel; color diffuse_color; color reflect_color; uint16_t scorebar; uint16_t score; bool won; bool lose; } full_state_t;
+#include "get_scene.h"
 #define K_gold_color	(fixed3_make(fixed_make_from_double(((double)1.2 * (double).15) * (double)243. / (double)256.), fixed_make_from_double(((double)1.2 * (double).15) * (double)201. / (double)256.), fixed_make_from_double(((double)1.2 * (double).15) * (double)104. / (double)256.)))
